tests/logitech_receiver/test_common.py

```diff
@@ -84,3 +206,40 @@ def test_int2bytes():
     result = common.int2bytes(value)
 
     assert result == expected
+
+
+def test_KwException():
```

Comment:
I'd rather stick to lower case names for test functions.

`def test_kw_exception():`

Reply:
done